src/main/kotlin/ink/abb/pogo/scraper/Bot.kt

```diff
@@ -94,11 +95,12 @@ class Bot(val api: PokemonGo, val settings: Settings) {
 
                 task(process)
                 task(hatchEggs)
+                task(evolve)
```

Comment:
How do you check for a null string if auto_evolve in config is empty?

Via != null or .length?  Strings can't be null in kotlin right?